bundle/tests/scorecard/config.yaml

```diff
@@ -79,7 +79,7 @@ stages:
   - entrypoint:
```

Comment:
This is where we patch to add the `--installOpenShiftCertManager` option?

Reply:
Right, after the first argument which is the executable name.